lib/grape-swagger/doc_methods/path_string.rb

```diff
@@ -2,7 +2,7 @@ module GrapeSwagger
   module DocMethods
     class PathString
       class << self
-        def build(path, options = {})
+        def build(path, route_version, options = {})
```

Comment:
It was my first thought https://github.com/ruby-grape/grape-swagger/pull/438/commits/eca701fad634b797824c9b9051deed1887860136 but the PathString specs would need then some complex route initialization.